tests/profiles/glyf_test.py

```diff
@@ -70,3 +70,18 @@ def test_check_glyf_non_transformed_duplicate_components():
     assert_results_contain(check(ttFont),
                            FAIL, 'found-duplicates')
 
+
+def test_check_glyf_nested_components():
+    """Check glyphs do not have nested components."""
+    check = CheckTester(opentype_profile,
+                        "com.google.fonts/check/glyf_nested_components")
+
+    ttFont = TTFont(TEST_FILE("nunito/Nunito-Regular.ttf"))
+    assert_PASS(check(ttFont))
+
+    # We need to create a nested component. "second" has components, so setting
+    # one of "quotedbl"'s components to "second" should do it.
+    ttFont['glyf']['quotedbl'].components[0].glyphName = "second"
+
+    assert_results_contain(check(ttFont),
+                           FAIL, 'found-duplicates')
```

Comment:
the code-test here is expecting the wrong message id.

...or the code implementation is emitting the wrong message id :-D 

<img width="640" alt="Captura de Tela 2020-11-09 às 11 48 58" src="https://user-images.githubusercontent.com/213676/98556238-96240280-2281-11eb-8176-0a8155e090d7.png">
https://travis-ci.org/github/googlefonts/fontbakery/jobs/741638322